webknossos/webknossos/dataset/dataset.py

```diff
@@ -225,10 +225,17 @@ def layers(self) -> Dict[str, Layer]:
         """
         return self._layers
 
-    def upload(self) -> str:
+    def upload(self, jobs: Optional[int] = None) -> str:
+        """
+        Uploads this dataset to webKnossos.
+
+        If supplied, the `jobs` parameter will determine the number of simultaneous chunk uploads.
```

Comment:
Warum nicht `If supplied, the `jobs` parameter will determine the number of simultaneous chunk uploads. Default  value 123.`?

Reply:
The downside would be that the default value is repeated in the docstring and could get out of sync with its actual definition. But it might be a valid trade-off. I don't have a strong opinion on this.